quickwit-indexing/failpoints/mod.rs

```diff
@@ -258,18 +258,15 @@ async fn test_merge_executor_controlled_directory_kill_switch() -> anyhow::Resul
 
     let metastore = test_index_builder.metastore();
     let splits_with_footer_offsets = metastore.list_all_splits(index_id).await?;
```

Comment:
we should rename all footer_offsets stuff. Can you grep for them?

Reply:
I think I found all places now (we also have split_id_and_footer_offsets)